sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp

```diff
@@ -48,13 +48,24 @@
 namespace autoware::pointcloud_preprocessor
 {
 
+struct AngleConversion
+{
+  // Equation for the conversion between sensor azimuth coordinates and Cartesian coordinates:
+  // sensor azimuth coordinates = offset_rad + sign * cartesian coordinates;
+  // offset_rad is restricted to be a multiple of 90, and sign is restricted to be 1 or -1.
+  float offset_rad{0};
+  float sign{1};
+  float offset_rad_threshold{0.087f};  // (5 / 180) * M_PI
+  float sign_threshold{0.1f};
+};
+
 class DistortionCorrectorBase
 {
 public:
   virtual bool pointcloudTransformExists() = 0;
   virtual bool pointcloudTransformNeeded() = 0;
-  virtual std::deque<geometry_msgs::msg::TwistStamped> get_twist_queue() = 0;
-  virtual std::deque<geometry_msgs::msg::Vector3Stamped> get_angular_velocity_queue() = 0;
+  virtual std::deque<geometry_msgs::msg::TwistStamped> getTwistQueue() = 0;
```

Comment:
As a general comment, the current Autoware coding style mandates `snake_case`, not `camelBack`. Please adhere to that style :pray:
* [.clang-tidy](https://github.com/autowarefoundation/autoware/blob/949a69e49ae8a0fb7d710080e5b613d926e13e21/.clang-tidy#L438-L439)
* [Autoware coding guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/#use-lower-snake-case-for-function-names-required-partially-automated)

Reply:
Thanks, I changed all of them to snake case in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/f6be6374320c33f9a3bea6e6f0f58fd9cc010d44